sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/ContactDirectorySteps.java

```diff
@@ -34,13 +35,27 @@
 
   protected WebDriverHelpers webDriverHelpers;
   private final AssertHelpers assertHelpers;
+  public static String LAST_CREATED_CONTACT_URL;
```

Comment:
Please convert the class variable to local variable

Reply:
reworked.